nca/CoreDS/DimensionsManager.py

```diff
@@ -24,11 +24,12 @@ class __DimensionsManager:
         def __init__(self):
             # TODO: verify alphabet for regex type dimensions, currently using one default alphabet
             #  currently valid chars are: ['.', '/', '-', 0-9, a-z, A-Z ]
-            self.default_dfa_alphabet_chars = ".\\w/\\-"
-            self.default_dfa_alphabet_str = "[" + self.default_dfa_alphabet_chars + "]*"
+            self.default_dfa_alphabet_str = MinDFA.default_alphabet_regex
+            self.default_dfa_alphabet_chars = MinDFA.default_dfa_alphabet_chars
```

Comment:
Are these two data members still needed? I guess `MinDFA` attributes can be used instead...

Reply:
Removed from `DimensionsManager`